app/templates/gentelella/admin/footer.html

```diff
@@ -85,4 +83,35 @@ <h4 class="footer-header">Use Open Event</h4>
         <div class="copyright">© 2016 Open Event</div>
     </div>
 </footer>
+<script>
+    $( document ).ready(function() {
+        var get_request_url = window.location.protocol + "//" + window.location.host + "/get_languages";
```

Comment:
What is the requirement for an AJAX call to get the list of languages ? You could load directly when the page is loaded.